modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java

```diff
@@ -278,6 +278,41 @@ public void testAddCustomObjectField() throws Exception {
 						true
 					).build())));
 
+		ObjectDefinition objectDefinition1 =
+			ObjectDefinitionTestUtil.addObjectDefinition(
+				_objectDefinitionLocalService);
+		ObjectDefinition objectDefinition2 =
+			ObjectDefinitionTestUtil.addObjectDefinition(
+				_objectDefinitionLocalService);
+
+		String objectRelationshipName = "a" + RandomTestUtil.randomString();
+
+		_objectRelationshipLocalService.addObjectRelationship(
+			TestPropsValues.getUserId(),
+			objectDefinition1.getObjectDefinitionId(),
+			objectDefinition2.getObjectDefinitionId(), 0,
+			ObjectRelationshipConstants.DELETION_TYPE_PREVENT,
+			LocalizedMapUtil.getLocalizedMap(RandomTestUtil.randomString()),
+			objectRelationshipName,
+			ObjectRelationshipConstants.TYPE_ONE_TO_MANY);
```

Comment:
We can simply this creating a new method in ObjectRelationshipTestUtil to receive the objectRelationshipName, something like this:
```java
public static ObjectRelationship addObjectRelationship(
		ObjectRelationshipLocalService objectRelationshipLocalService,
		ObjectDefinition objectDefinition1,
		ObjectDefinition objectDefinition2,
                String objectRelationshipName)
	throws PortalException {
```